cmd/libsnap-confine-private/snap-test.c

```diff
@@ -214,11 +214,94 @@ static void test_sc_snap_name_validate__respects_error_protocol(void)
 	    ("snap name must use lower case letters, digits or dashes\n");
 }
 
+static void test_sc_snap_drop_instance_name_no_dest(void)
```

Comment:
It is fine as is but it might be nice if the test names used '_base' instead of '_dest' since that is more inline with the implementation.

Reply:
I meant `dest` as in _destination buffer_. Let me rename it.